app/dataviz/templates/dataviz/circles.html

```diff
@@ -0,0 +1,153 @@
+{% extends "admin/base_site.html" %}
```

Comment:
We'll want to strip some of the whitespace and additional lines throughout the new template files.

Reply:
there a programatic way to do it?  `make fix` isnt seeming to do the trick